main/create_simulated_body_model.py

```
@@ -0,0 +1,1085 @@
+    # ---- Selection / drag-move ----
+    def _select_model_at(self, img_v: float, img_u: float) -> None:
+        # Check bodies first: order_near_to_far; pick first whose mask contains pixel
+        body_masks = self._last_meta.get('body_masks', [])
+        order_near_to_far = self._last_meta.get('order_near_to_far', [])
+        bodies = self.sim_params.get('bodies', [])
+        if body_masks and bodies:
+            # Map names to index in original list by matching 'name' upper
+            name_to_index = {b.get('name', '').upper(): i for i, b in enumerate(bodies)}
+            height = int(self.sim_params['size_v'])
+            width = int(self.sim_params['size_u'])
+            v_i = int(round(img_v))
```

Ruff (0.14.5)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)